cellprofiler/pipeline.py

```diff
@@ -1287,6 +1287,23 @@ def savetxt(self, fd_or_filename,
             write_file_list(fd, self.__file_list)
         if needs_close:
             fd.close()
+    
+    def save_pipeline_notes(self, fd, indent = 2):
```

Comment:
Why haven’t you synchronized and closed the file? Does this happen elsewhere? If so, why?